students/10th/soojungsohn/user/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import (
+        SignUpView,
```

Comment:
기본적으로는 줄바꿈 후에 4칸만 들여쓰기 하도록 수정 부탁드립니다.
https://www.notion.so/wecode/Wecode-Python-Coding-Convention-0bd47a6018634454b9ee1e27984cb83a
```suggestion
    SignUpView,
```